code-studio/package.json

```diff
@@ -74,7 +74,8 @@
     "linklocal": "^2.5.2",
     "lodash": "^3.10.1",
     "marked": "0.3.2",
-    "react": "^0.13.1",
+    "react": "^0.14",
+    "react-dom": "^0.14",
```

Comment:
Again, probably pin this to 0.14.7?  Also, isn't it scary that we were using 0.13.3 in our tests but running 0.13.1 through dashboard?